runbotics/runbotics-orchestrator-ui/src/theme/light.ts

```diff
@@ -1,8 +1,19 @@
 import { colors, ThemeOptions } from '@mui/material';
+import { palette } from '@mui/system';
 
 import { softShadows } from './shadows';
 
-export const lightTheme: ThemeOptions = {
+type InfoIconPalette = ThemeOptions['palette'] & {
+    infoIcon?: {
+        default: string
+    }
+}
+interface ExtendedThemeOptions extends Omit<ThemeOptions, 'palette'> {
```

Comment:
that's not how you do it
see this https://mui.com/material-ui/customization/theming/#custom-variables
then that `runbotics\runbotics-orchestrator-ui\src\@types\mui.d.ts`

we do it this way so the type for theme accessible inside mui components is adjusted as well and everything is type safe

Reply:
the best place for it is `theme.palette.info.icon` but I tried it before and it requires a walkaround so you can go with just a new field like `theme.process.action.infoIcon` yolo